MAINTAINERS.yml

```diff
@@ -809,6 +811,8 @@ Devicetree:
   collaborators:
     - decsny
     - galak
+  files-regex:
+    - dts/bindings/.*zephyr.*
```

Comment:
Not sure why any binding with zephyr in it should belong to this area?

Reply:
why wouldn't they? those types of bindings should be reviewed by devicetree maintainers, as they should not be added lightly

Reply:
in the past all the DT bindings were reviewed by DT maintainers but now with the recent change I dont get added to review any bindings, and apparently now we have to rely on the review of the other area maintainers to review the DT usage properly, and honestly, I cannot say that I trust all the other maintainer of different areas to have the same type of glasses on when reviewing DT bindings. And what makes this more annoying is that nobody consulted me about this change to this DT binding area even though I am the only active one on the list (did somebody ask Kumar before making this change at least?) Regardless I am exhausted already from reviewing so many PRs due to that filter, so I am okay to limit it just to ones that have `zephyr` in the name